File: swift练习/task 10/app2/app2/ViewController.m
//
//  ViewController.m
//  app2
//
//  Created by 张皓 on 16/7/12.
//  Copyright © 2016年 张皓. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *showText;

@end

@implementation ViewController
- (IBAction)openApp1:(id)sender {
    
    NSString *strUrl = @"app1://appname=app2";
    
    NSURL *url = [NSURL URLWithString:strUrl];
    
    BOOL b = [[UIApplication sharedApplication] canOpenURL:url];
    
    if (b) {
        [[UIApplication sharedApplication] openURL:url];
    }
    
    
}

- (void)viewDidLoad {
    [super viewDidLoad];
 
    UIPasteboard *board2 = [UIPasteboard pasteboardWithName:@"app1_board" create:NO];
    
    self.showText.text = board2.string;
}



@end

File: swift练习/11/userDefault/userDefault/ViewController.swift
//
//  ViewController.swift
//  userDefault
//
//  Created by 张皓 on 16/8/2.
//  Copyright © 2016年 张皓. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var ud: NSUserDefaults!
    
    
    @IBAction func saveBtnPress(sender: AnyObject) {
        
        
        ud.setObject(inputText.text, forKey: "data")
        
        print("Saved")

        
    }
    @IBOutlet var inputText: UITextView!
    override func viewDidLoad() {
        super.viewDidLoad()
        
        ud = NSUserDefaults.standardUserDefaults()
        
        if let value = ud.objectForKey("data") {
            inputText.text = value as! String
        }
        
        
        
        
    }

   
}

